subprojects/android-test/ui-testing-core/src/main/kotlin/com/avito/android/test/UITestConfig.kt

```diff
@@ -53,6 +54,13 @@ object UITestConfig {
         AssertionError::class.java
     )
 
+    /**
+     * Pattern for launcher package in Android 11+ devices.
+     * Hardcoded values due to package visibility restrictions:
+     * https://developer.android.com/training/package-visibility/declaring
+     */
+    var deviceLauncherPackage: Pattern = Pattern.compile(".+launcher.*")
```

Comment:
I've extracted this hardcode to config.
It may be useful to fix it in local builds before fixing in the test library.